src/routes/user/updateUser.ts

```diff
@@ -0,0 +1,81 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { knexController } from '../../database/database';
+import { User, editUserResponse, user } from '../../types/user.types';
+
+import bcrypt from 'bcrypt';
+
+const editUserSchema = {
+  description: 'Edit data of already existing user.',
+  summary: 'Edit user',
+  tags: ['User'],
```

Comment:
Mikä käytäntö näiden tägien kanssa on? Olisko esim. tässä EditUser, User vai joku muu?

Reply:
Nää tägit kertoo Swaggerille, minkä otsikoiden alle eri endpointit lajitellaan. Näitä voi tägäillä isompina kokonaisuuksina, esim just User tagin alle. Tageja voi myös antaa useita, niin voidaan lajitella esim kaikki update endpointit saman tagin alle jne